jsumfields.php

```diff
@@ -2150,7 +2151,7 @@ function _jsumfields_generate_data_based_on_current_data($session = NULL) {
     $session = CRM_Core_Session::singleton();
   }
   if (empty($sumfieldsCustomTableName)) {
-    $session::setStatus(ts("Your configuration may be corrupted. Please disable and renable this extension."), ts('Error'), 'error');
+    $session::setStatus(ts("Your configuration may be corrupted. Please disable and renable this extension."), E::ts('Error'), 'error');
```

Comment:
@lucky091588 Looks like there's one more `ts()` that needs attention here. Please fix?